Fixed #34388 -- Allowed using choice enumeration types directly on model and form fields. (#16629)
django/django: docs/releases/5.0.txt

  of ``ValidationError`` raised during
  :ref:`model validation <validating-objects>`.

> * :attr:`Field.choices <django.db.models.Field.choices>` now also accepts

Reviewer: I'd chop "also".

>   ``Choices`` classes directly instead of requiring manual expansion
>   through ``Choices.choices``.

Reviewer 2: ```suggestion
  ``Choices`` classes directly instead of requiring expansion with
  the ``choices`` attribute.
```
